tests/src/test/scala/system/health/BasicHealthTest.scala

```diff
@@ -109,7 +120,7 @@ class BasicHealthTest
                     }
 
                 println("Polling for activations")
-                val activations = wsk.activation.pollFor(N = 1, Some(triggerName), retries = 30)
+                val activations = wsk.activation.pollFor(N = 5, Some(triggerName), since = Some(start), retries = 30)
```

Comment:
It is a completely arbitrary number > 1

I picked 5 because it seemed like the largest reasonable number of activations that could be caused by simultaneous tests posting a message to this same topic. Nevermind that such messages would likely get batched into one invoke... 5 seemed sufficiently large.